tensorflow_datasets/core/dataset_info.py

```diff
@@ -673,14 +691,14 @@ def load_metadata(self, data_dir):
 class BeamMetadataDict(MetadataDict):
   """A `tfds.core.Metadata` object supporting Beam-generated datasets."""
 
-  def __init__(self, *args, **kwargs):
+  def __init__(self, *args: Any, **kwargs: Any):
     super(BeamMetadataDict, self).__init__(*args, **kwargs)
     self._tempdir = tempfile.mkdtemp("tfds_beam_metadata")
 
-  def _temp_filepath(self, key):
+  def _temp_filepath(self, key: str) -> str:
     return os.path.join(self._tempdir, "%s.json" % key)
 
-  def __setitem__(self, key, item):
+  def __setitem__(self, key: str, item: beam.pvalue.PValue) -> Optional[str]:
```

Comment:
Everywhere beam is used, it should be wrapped in `"` (`: "beam.pvalue.PValue"`)

Also return type should be `None`

Reply:
Done